requirements.txt

```diff
@@ -2,6 +2,8 @@ aiohttp==3.7.4.post0
 cryptography==3.4.6
 PyMySQL==1.0.2
 pg8000==1.18.0
+# python-tds==1.10.0
+git+https://github.com/denisenkom/pytds.git@refs/pull/120/merge
```

Comment:
I think I've mentioned this before, but we should probably make these scoped requirements rather than include them all by default.

Reply:
just updated the PR to scope the dependencies in setup.py